### PR TITLE
[ERE-929] Allow multiple clinicians per patient

### PR DESCRIPTION
**Pull request by alintreznai**

### CODE REVIEW

File: rdrf/registry/patients/admin_forms.py [hunk 1]
```diff
@@ -333,14 +338,24 @@ def __init__(self, *args, **kwargs):
...
+        self.fields["registered_clinicians"].label_from_instance = clinician_display_str
 
-        # clinicians field should only be visible for registries which
+        # registered_clinicians field should only be visible for registries which
         # support linking of patient to an "owning" clinician
+        wgs_set_by_clinicians = False
         if self.registry_model:
             if not self.registry_model.has_feature(RegistryFeatures.CLINICIANS_HAVE_PATIENTS):
-                self.fields["clinician"].widget = forms.HiddenInput()
+                self.fields["registered_clinicians"].required = False
```
Comment (ScriptSmith) [inline rdrf/registry/patients/admin_forms.py]: I don't think it should be required. In MND's case, patients can self-register without their clinician, same for Angelman.

Comment (alintreznai) [reply rdrf/registry/patients/admin_forms.py]: I think it's required by default that's why I set it to False if the clinicians_have_patients feature is not enabled.

Comment (ScriptSmith) [reply rdrf/registry/patients/admin_forms.py]: Previously it was:

```
clinician = models.ForeignKey(
        CustomUser,
        blank=True,
        null=True,
        verbose_name=_("Clinician"),
        on_delete=models.SET_NULL)
```

 `blank` was removed in this branch. I should've made the comment in the model.

Comment (alintreznai) [reply rdrf/registry/patients/admin_forms.py]: Updated.

**Pull request merged**